packages/core/src/abstract-dialect/query-generator.types.ts

```
@@ -1,9 +1,11 @@
+import type { AbstractDataType } from '../abstract-dialect/data-types.js';
```

ESLint

[error] Useless path segments for "../abstract-dialect/data-types.js", should be "./data-types.js"

(import/no-useless-path-segments)